tools/topology/sof-tgl-rt711-rt1308.m4

```diff
@@ -20,22 +20,48 @@ include(`common/tlv.m4')
 # Include Token library
 include(`sof/tokens.m4')
 
-include(`platform/intel/tgl.m4')
+include(`platform/intel/'PLATFORM`.m4')
 
 # Define pipeline id for intel-generic-dmic.m4
 # to generate dmic setting
 
-ifelse(CHANNELS, `0', ,
+ifdef(`AMPS',
```

Comment:
Can we have a comment here explaining AMPS so that subsequent users will know how to use this option. Also is this specific to any type of amplifier or is it generic ?

Reply:
> Can we have a comment here explaining AMPS so that subsequent users will know how to use this option. Also is this specific to any type of amplifier or is it generic ?

OK. I will add the comment in the next version.